dist/src/main/resources/bin/windup

```diff
@@ -165,6 +165,10 @@ if [ -r "$WINDUP_HOME/addons/" ] ; then
   ADDONS_DIR="--immutableAddonDir $WINDUP_HOME/addons/"
 fi
 
+if [ -z "$MAX_MEMORY" ] ; then
+  MAX_MEMORY="2048m"
+fi
+
```

Comment:
I'm still -1 on this patch (for reasons that I have mentioned previously), but I definitely wouldn't recommend going larger. Swapping and GC'ed languages combine extremely poorly for various reasons, and there are still a lot of 4GB languages out there.